GNUmakefile

```diff
@@ -43,7 +43,12 @@ bin/terraform:
 	(cd $(CURDIR)/bin/ ; unzip terraform.zip)
 
 testacc: fmtcheck bin/terraform
-	PATH="$(CURDIR)/bin:${PATH}" TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=90s
+	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=90s
+
+# Commenting out the below because it tries to use an amd64 version of terraform that is incompatible with M1 Macs. Instead,
+# simply rely on the version of terraform that's already on your path. If you don't have terraform installed, follow instructions
+# here to install https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
+# PATH="$(CURDIR)/bin:${PATH}" TF_ACC=1 go test $(TEST) -v $(TESTARGS) -run "TestTIDBResourceGroup_basic" -timeout=90s
```

Comment:
Do we need this as a separate section in Makefile or can we remove it since I've patched the ARCH problem?

Reply:
I'm removing it since it's patched for ARCH above.

LMK if you disagree

Reply:
Sounds good! Thanks for fixing this 👍